File: Programming Basic/Conditional_statements_exercise/02_Bonus_Score.py
number = float(input())

if number <= 100:
    bonus = 5
elif number > 1000:
    bonus = 0.1 * number
else:
    bonus = 0.2 * number

if number % 2 == 0:
    bonus += 1
elif number % 10 == 5:
    bonus += 2

print(bonus)
print(bonus + number)

File: Java Script Basic - April 2020/Conditional_Statements_Exercise/02_Bonus_Score.js
function bonusScore(score) {
    score = Number(score)
    let bonus = 0.0

    if (score <= 100) {
        bonus = 5
    } else if (score > 1000) {
        bonus = score * 0.1
    } else {
        bonus = score * 0.2
    }

    if (score % 2 === 0) {
        bonus += 1
    } else if (score % 10 === 5) {
        bonus += 2
    }

    let result = bonus + score

    console.log(bonus)
    console.log(result)


}

bonusScore(15875)